docs/usage.rst

```diff
@@ -27,9 +24,8 @@ segmentation fault. The docker image is set up to use this setting by default in
 Running the model
 -----------------
 
-Basic model operation can be seen in this short, self-explanatory example::
+Basic model operation can be seen in this short, mostly self-explanatory example::
 
-    from mpi4py import MPI
```

Comment:
Just curious where the `mpi4py` import moved, into the wrapper?

Reply:
Yes, it was also in the wrapper before.